doc/help.rst

```diff
@@ -0,0 +1,18 @@
+Useful Links
+============
+
+Cube Browser notebooks make use of a number of Python packages and projects.
+In most cases, you will be able to find help and information from the relevant website.
+Links to the websites of these projects are listed below:
+
+-  `Anaconda <https://anaconda.org/>`_
```

Comment:
@corinnebosley did you mean this link to point to the Anaconda cloud (i.e the place where Conda users can upload their conda packages to and other users can go to get these packages) or did you mean [Continuum's website for Anaconda](https://www.continuum.io/downloads)?